eth/catalyst/simulated_beacon.go

```diff
@@ -302,7 +302,7 @@ func (c *SimulatedBeacon) AdjustTime(adjustment time.Duration) error {
 		return errors.New("parent not found")
 	}
 	withdrawals := c.withdrawals.gatherPending(10)
-	return c.sealBlock(withdrawals, parent.Time+uint64(adjustment))
+	return c.sealBlock(withdrawals, parent.Time+uint64(adjustment/1e9))
```

Comment:
Nitpick, please change `1e9` to `time.Second`

Reply:
```suggestion
	return c.sealBlock(withdrawals, parent.Time+uint64(adjustment/time.Second))
```